applications/dashboard/controllers/class.entrycontroller.php

```diff
@@ -1106,7 +1110,9 @@ public function signIn($method = false, $arg1 = false) {
                                 Gdn::userModel()->setField(val('UserID', $user), ['Password' => $pw, 'HashMethod' => 'Vanilla']);
                             }
 
+                            Gdn::userModel()->fireEvent('BeforeSignIn', ['UserID' => $user->UserID]);
                             Gdn::session()->start(val('UserID', $user), true, (bool)$this->Form->getFormValue('RememberMe'));
+                            Gdn::userModel()->fireEvent('AfterSignIn');
```

Comment:
I don't think it's necessary to move this event, so let's drop it back where it originally was.